assets/js/atomic/blocks/product-elements/title/frontend.js

```diff
@@ -10,4 +9,4 @@ import withFilteredAttributes from '@woocommerce/base-hocs/with-filtered-attribu
 import Block from './block';
 import attributes from './attributes';
 
-export default compose( withFilteredAttributes( attributes ) )( Block );
```

Comment:
Since `@wordpress/compose` is no longer extracted, its dependency was generating a new frontend file specific for vendor files. Taking a closer look, it seems like `compose()` is not required here, so let's remove it.